lib/plugins/boluswizardpreview.js

```diff
@@ -124,7 +125,20 @@ function init() {
     results.effect = iob * profile.getSensitivity(sbx.time);
     results.outcome = scaled - results.effect;
     var delta = 0;
+
+    var timeSinceLastMeal = 1000000000;
+    var now = (new Date).getTime();
     
+    for (var i = 0; i < sbx.data.treatments.length; i++) {
+      var treatment = sbx.data.treatments[i];
+      var delta = now - treatment.mills;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Variable 'delta' is already declared in line 127.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/944034239/issues/source?bid=2452379&fileBranchId=2847953#l134)